spreadsheet_oca/static/src/spreadsheet/bundle/spreadsheet.xml

```diff
@@ -386,4 +386,43 @@
             </div>
         </div>
     </t>
+    <t t-name="spreadsheet_oca.MenuSelector" owl="1">
+  </t>
+  <t t-name="spreadsheet_oca.OdooPanel" owl="1">
+    <div class="o-section">
+        <div class="o-section-title">Link to Odoo menu</div>
+        <div class="o-section-value">
+            <Many2OneField
+                    value="menuId"
+                    relation="'ir.ui.menu'"
+                    record="record"
+                    update.bind="updateMenu"
+                    canOpen="false"
+                />
```

Comment:
Incorrect indentation

Reply:
Pre-commit is forcing this configuration :sob:

Reply:
But that happened usually because there was already a bad indentation somewhere, and once with it, the ugly format is respected. Try to check whole file and put it correctly manually, and you'll see it's respected.

Reply:
You were totally right. A space was misconfiguring everything :cry: